### DISCUSSION TITLE
Understanding different log levels

### DISCUSSION THREAD

**Discussion by carloswm85** [Bungeefan/logger#82] [Q&A] [answered]
https://pub.dev/documentation/logger/latest/logger/Level.html

As far as I know, some log levels have more hierarchy than other, so when filtering them out, some will pass while other will not. In the case of your logger, what's that hierarchy? Do I make the filtering only by using the LogFilter class?

**Comment by Bungeefan** [comment, marked as answer]
> what's that hierarchy?

Hi, I am not quite sure what you mean by that. As you can see, each LogLevel has a number assigned, and that gives it its position/priority in the default filters.

> Do I make the filtering only by using the LogFilter class?

Yes, that would be the plan.

**Comment by carloswm85** [comment]
Ok. Understood.
